app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -19,61 +19,103 @@ inset_text:
+Most teacher training courses are non-salaried. This means that you have to pay course fees and will not earn a salary while you train.
 
-### Teach First
-Teach First is a 2-year programme leading to a PGDE and QTS. At the end of your first year, you will have QTS. The PGDE is awarded at the end of the second year.
+Fees are usually around £9,250, and can be higher for [international students](/non-uk-teachers/train-to-teach-in-england-as-an-international-student).
+
+If you train to teach certain secondary subjects, you may be eligible for a [scholarship or bursary](/funding-and-support/scholarships-and-bursaries).
+
+No matter what course you choose, you can apply for a student loan.
+
+[Learn about how you could fund your teacher training](/funding-and-support).
+
+### Teacher training providers
+
+Your teacher training course might be provided by a university, school, college or group of schools. Your course might be referred to as:
+
+* school-centred initial teacher training (SCITT)
+* School Direct (there are salaried and non-salaried School Direct courses)
+
+The content of SCITT and School Direct programmes are broadly the same.
+
+Some providers have different course structures and have placements in different kinds of schools, such as special schools. Talk to your provider to find out what they can offer you.
+
+[Find teacher training providers](https://www.find-postgraduate-teacher-training.service.gov.uk/).
+
+### Choose your provider
 
-To apply for Teach First, you should [visit the Teach First website](https://www.teachfirst.org.uk/).
+One of the best ways to decide between providers is to contact them.
 
-### Postgraduate teaching apprenticeship
+Ask them what a typical day looks like, how you’ll be supported, and if they have any open days or [teacher training events](/events) you can attend.
+
+### Find out where you’ll spend your time
+
+You should talk to teacher training providers about where your school placements and theoretical learning will be. The address of your teacher training provider may not be where you spend most of your time. For example, your teacher training provider might be a university, but you’ll spend most of your time in placement schools.
+
+[Find out more about what your teacher training will be like](/train-to-be-a-teacher/initial-teacher-training).
+
+## Salaried teacher training
+
+Salaried courses are sometimes available for specific candidates, such as if you’re working as an unqualified teacher or teaching assistant. These include School Direct (salaried) courses and postgraduate teaching apprenticeships.
+
+These are in high demand and very competitive. Places fill up quickly.
+
+You should check your eligibility with the teacher training provider before you apply.
+
+[Find salaried teacher training](https://www.find-postgraduate-teacher-training.service.gov.uk/) by applying the ‘only show courses that come with a salary’ filter.
```

Comment:
From an accessibility perspective, this link text is slightly misleading as it implies that clicking on that link will take you to a pre-filtered set of results - people using screen readers could open the link without reading the rest of the sentence. Is there anyway we could reword this (I know it is difficult!)